cmd/action/ci/ci.go

```diff
@@ -18,10 +18,13 @@ package ci
 
 import (
 	"fmt"
+	"math/rand" //nolint:all
```

Comment:
Do we want to add a comment as to why this is done?
```suggestion
	"math/rand" //nolint:gosec  // We use randomization for appending unique suffixes and this need not be cryptographically secure.
```